src/main/java/com/revlearn/team1/controller/TransactionController.java

```diff
@@ -1,10 +1,43 @@
 package com.revlearn.team1.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.revlearn.team1.dto.TransactionDTO;
+import com.revlearn.team1.service.TransactionServiceImp;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 @RequestMapping("/api/v1/transaction")
 public class TransactionController
 {
+    @Autowired
+    TransactionServiceImp transactionService;
+
```

Comment:
Add your custom exception to the GlobalExceptionHandler file.  It has the "RestControllerAdvice" annotation, which is what enables you to return a particular response status, and message.